src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java

```diff
@@ -1276,9 +1276,9 @@ public Object validateStudySubject(SubjectDataBean subjectDataBean, Study tenant
             }
         }
         if (studySubject != null && !studySubject.getStatus().equals(Status.AVAILABLE)) {
-            //if participant has already signed, show they are not available instead of not found
+            
             if (studySubject.getStatus().equals(Status.SIGNED)) {
```

Comment:
merge this if statement with the one above this.

Reply:
done